polaris-react/src/components/Navigation/components/Item/tests/Item.test.tsx

```diff
@@ -486,7 +508,7 @@ describe('<Nav.Item />', () => {
 
       expect(item).toContainReactComponent('a', {
         'aria-expanded': false,
-        'aria-controls': ':r18:',
+        'aria-controls': expect.stringMatching(/^:r\d+:$/),
```

Comment:
Previously, the value in this unit test had to be incremented each time new tests were added before it. For instance, it would need to be updated to `:r19:` now. However, a unit test should not rely on its position in the test suite to pass. To address this issue, the updated test will match any digit within the specified pattern.